app/uk/gov/hmrc/agentservicesaccount/controllers/AgentServicesController.scala

```diff
@@ -115,7 +115,7 @@ class AgentServicesController @Inject()
     else Future.successful(())
   }
 
-  val yourAccount: Action[AnyContent] = Action.async { implicit request =>
+  val yourAccount: Action[AnyContent] = actions.authActionCheckSuspend.async { implicit request =>
     withFullUserDetails { (agentInfo: AgentInfo) =>
```

Comment:
do you still need this...I thought you get agentInfo with the authActionCheckSuspend?

Reply:
or actually, there is some duplication here but I thin you don't have email or name with the authActionCheckSuspend

Reply:
Yes you are correct, I don't actually need this